designsystem/src/main/res/layout/ds_menu_view.xml

```diff
@@ -59,20 +56,36 @@
 
     <androidx.appcompat.widget.AppCompatTextView
         android:id="@+id/ds_menu_label"
-        android:layout_width="0dp"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/ds_submenu_text_margin_top_bottom"
         android:layout_marginEnd="@dimen/ds_spacin_tiny"
-        android:layout_marginRight="@dimen/ds_spacin_tiny"
+        android:layout_marginStart="@dimen/ds_spacing_standard"
         android:layout_marginBottom="@dimen/ds_submenu_text_margin_top_bottom"
+        app:layout_constrainedWidth="true"
         android:gravity="center_vertical"
         android:textColor="@color/colorBrdNatGray"
         android:textSize="@dimen/ds_text_small"
+        app:layout_constraintHorizontal_bias="0.0"
         app:layout_constraintBottom_toBottomOf="@+id/ds_menu_icon"
-        app:layout_constraintEnd_toStartOf="@+id/ds_menu_arrow"
+        app:layout_constraintEnd_toStartOf="@+id/ds_menu_tag"
         app:layout_constraintStart_toEndOf="@+id/ds_menu_icon"
         app:layout_constraintTop_toTopOf="@+id/ds_menu_icon"
-        tools:text="Sub Menu item" />
+        app:layout_constraintHorizontal_chainStyle="packed"
+        tools:text="Sub Menu item with a big name to test what happens" />
+
+    <com.natura.android.tag.Tag
+        android:id="@+id/ds_menu_tag"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="@dimen/ds_spacin_tiny"
```

Comment:
aqui a mesma ideia, o mellhor seria usar` ?spacingTiny`.
fiquei pensando tbm se a gnt define o `marginStart` pra gnt poder garantir que a tag não vai ficar coladona no texto do menu se ele for grandão...